tools/_get_expiration.js

```diff
@@ -0,0 +1,93 @@
+var conf = require('../loop/config').conf;
+var async = require('async');
+var redis = require("redis");
+
+var storage = conf.get("storage");
+
+var args = process.argv.slice(2);
+
+var verbose = args.indexOf('--verbose') !== -1;
```

Comment:
Should we use something like https://www.npmjs.com/package/argparse to ease arguments for the CLI ?